src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__schoolmint_grow_observation_details.sql

```diff
@@ -154,126 +155,337 @@ with
+                    s.form_short_name,
+                    s.internal_id,
+                    o.score_measurement_id
+                order by o.observed_at desc
+            ) as rn_submission,
+        from scaffold as s
+        left join
+            observations as o
+            on cast(o.observed_at as date) between s.start_date and s.end_date
+            /* Matches on name for PM Rounds to distinguish Self and Coach */
+            and s.form_type = 'PM'
+            and s.form_short_name = o.form_short_name
+            and s.user_id = o.teacher_id
 
-union all
+        union all
+
+        select
+            s.user_id,
+            s.role_name,
+            s.internal_id,
+            s.form_type,
+            s.form_term,
+            'ETR + S&O' as score_type,
+            s.form_short_name,
+            s.start_date,
+            s.end_date,
+            s.academic_year,
+            s.employee_number,
+            s.teammate,
+            s.entity,
+            s.location,
+            s.grade_band,
+            s.home_work_location_powerschool_school_id,
+            s.department,
+            s.grade_taught,
+            s.job_title,
+            s.manager,
+            s.worker_original_hire_date,
+            s.assignment_status,
+
+            o.observation_id,
+            o.teacher_id,
+            o.form_long_name,
+            o.created,
+            o.observed_at,
+            o.observer_name,
+            o.overall_score,
+            o.glows,
+            o.grows,
+            o.score_measurement_id,
+            o.score_percentage,
+            o.row_score_value,
+            o.measurement_name,
+            o.measurement_scale_min,
+            o.measurement_scale_max,
+            o.tier,
+            o.text_box,
+
+            row_number() over (
+                partition by
+                    s.form_type,
+                    s.form_short_name,
+                    s.internal_id,
+                    o.score_measurement_id
+                order by o.observed_at desc
+            ) as rn_submission,
+
+        from scaffold as s
+        left join
+            observations as o
+            on cast(o.observed_at as date) between s.start_date and s.end_date
+            /* matches only on type and date for weekly forms */
+            and s.form_type in ('WT', 'O3')
+            and s.form_type = o.form_type
+            and s.user_id = o.teacher_id
+    ),
+
+    historical_overall_scores as (
+        select
+            employee_number,
+            academic_year,
+            pm_term as form_term,
+            etr_score,
+            so_score,
+            overall_score,
+            etr_tier,
+            so_tier,
+            overall_tier as tier,
+        from
+            {{
+                source(
+                    "performance_management",
+                    "src_performance_management__scores_overall_archive",
+                )
+            }}
+
+        union distinct
```

Comment:
what is causing duplicates that requires the use of `distinct`? is there a data model problem we need to solve first?

Reply:
it's that every row in observation_details has an overall_score that is identical, and it's one row per measurement. I just need the overall_score once.